src/components/30-organisms/modal/CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+## 4.0.10
+
+Change mouse close handler event listener.
```

Comment:
Why change? What bug ticket motivates this? This line needs a `#<github issue number> at the end!